Dockerfile

```diff
@@ -1,44 +1,37 @@
-# syntax=docker/dockerfile:1.2
-# Stream swaps to CentOS Stream once and is reused.
-# FROM quay.io/centos/centos:stream8 AS stream
-FROM registry.access.redhat.com/ubi8/ubi:latest as base
-RUN set -ex\
-	; dnf install -y gpgme-devel \
-	; dnf -y -q clean all
-
-RUN dnf install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False git perl wget make gcc
-
-
-FROM base AS build
-
-ARG BUILDER_SRC
-
-ENV GO_VERSION=1.16.13
-ENV GO_OS=linux
-ENV GO_ARCH=amd64
-ENV GOPATH=/go
-
-RUN curl https://dl.google.com/go/go${GO_VERSION}.${GO_OS}-${GO_ARCH}.tar.gz --output go.tar.gz
-RUN tar -C /usr/local -xzf go.tar.gz > /dev/null
-ENV PATH=$PATH:/usr/local/go/bin:${GOPATH}/bin
+FROM registry.redhat.io/rhel8/go-toolset:1.16.12-4 as build
+USER root
+RUN dnf install -y --setopt=tsflags=nodocs git
+COPY . /go/src/
+RUN cd /go/src/ && env GOOS=linux GOARCH=amd64 make build
 
-RUN go version
 
-COPY . /go/src/${BUILDER_SRC}
-RUN cd /go/src/${BUILDER_SRC} && go mod vendor && make build
-
-
-FROM base AS final
+FROM registry.redhat.io/ubi8/podman:8.5-15
 LABEL maintainer "Quay devel<quay-devel@redhat.com>"
 
-ARG BUILDER_SRC
-
-COPY --from=build /go/src/${BUILDER_SRC}/bin/quay-builder /usr/local/bin
+RUN set -ex\
+	; dnf install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False git wget \
+	; dnf -y -q clean all
 
+COPY --from=build /go/src/bin/quay-builder /usr/local/bin
 COPY buildpack/ssh-git.sh /
-ADD load_extra_ca.rhel.sh /load_extra_ca.sh
-ADD entrypoint.sh /entrypoint.sh
+COPY entrypoint.sh /home/podman/entrypoint.sh
+
+# Rootless/unprivileged buildah configurations
+# https://github.com/containers/buildah/blob/main/docs/tutorials/05-openshift-rootless-build.md
+RUN touch /etc/subgid /etc/subuid && \
+    chmod g=u /etc/subgid /etc/subuid /etc/passwd && \
+    echo 'podman:100000:65536' > /etc/subuid && echo 'podman:100000:65536' > /etc/subgid && \
```

Comment:
Not familiar with this mapping. Does this mean uid 0 in the container will be uid 10000 on the host?
Any particular reason for this specific range?